World/Updates/Rel22/Rel22_04_063_tekton_scripting.sql

```diff
@@ -43,10 +43,19 @@ BEGIN
         -- -- PLACE UPDATE SQL BELOW -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- --
         -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
 
+-- Remove any pre-existing strings/scripts related to Tekton and Agee Tyler.
+DELETE FROM `db_script_string` WHERE `content_default` IN ('HEY! What the heck, guy?', 'Of all the slimy, inconsiderate, bullheaded...');
+DELETE FROM `db_scripts` WHERE `id` = 48873;
+
+-- Remove GUIDs in creature table between 400,050 and 400,073.
+DELETE FROM `creature` WHERE `guid` IN (400050, 400051, 400052, 400053, 400054, 400055, 400056, 400057, 400058, 400059, 400060, 400061, 400062, 400063, 400064, 400065, 400066, 400067, 400068, 400069, 400070, 400071, 400072, 400073);

```

Comment:
If you like too, you can use the **BETWEEN** macro if you don't want to write out of the **guids**.
e.g.
```sql
DELETE FROM `creature` WHERE `guid` BETWEEN 400050 AND 400073;
```